src/groundlight/binary_labels.py

```diff
@@ -28,11 +35,11 @@ def convert_internal_label_to_display(
 ) -> str:
     # NOTE: Someday we will do nothing here, when the server provides properly named classes.
     upper = label.upper()
-    if upper == "PASS":
-        return "YES"
-    if upper == "FAIL":
-        return "NO"
-    if upper in ["YES", "NO"]:
+    if upper == PASS_DEPRECATED:
```

Comment:
Is this right? I thought that 0 == Pass == No and 1 == FAIL == Yes

Reply:
Based on the way the prior code was set up (and the diagram I wrote to myself because I keep forgetting and this mapping is terrible), I think it's correct:

0 == PASS == YES == Normal
1 == FAIL == NO == Anomalous

![IMG_1588](https://user-images.githubusercontent.com/4020875/233230904-2e4ac9b1-2cb2-4350-8985-1745e8c84ac2.jpg)

Reply:
Also, https://github.com/positronix-ai/zuuul/blob/main/janzu/apparati/views.py#L1178 says PASS == YES and FAIL == NO